llarp/util/decaying_hashset.hpp

```diff
@@ -56,13 +48,36 @@ namespace llarp
         return m_CacheInterval;
       }
 
+      bool
+      Empty() const
```

Comment:
maybe IsEmpty() ?

Shouldn't be too ambiguous where it's used, but there is a nonzero chance it could be confused with the verb "empty".

Reply:
(though the stl standard seems to be that empty() is a query, not a command, so I guess this is fine)